### PR TITLE
Model change to use GPU properly

### PR DESCRIPTION
**Pull request by jasonkneen**
## Summary by Sourcery

Change the face swapper model to use a non-fp16 ONNX file for better GPU support.

Enhancements:
- Update the model file used for face swapping to a non-fp16 version to improve GPU compatibility.

### CODE REVIEW

File: modules/processors/frame/face_swapper.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 def pre_check() -> bool:
     download_directory_path = resolve_relative_path('../models')
-    conditional_download(download_directory_path, ['https://huggingface.co/hacksider/deep-live-cam/blob/main/inswapper_128_fp16.onnx'])
```
[the remark this answers is not in the document]
Comment (jasonkneen) [reply modules/processors/frame/face_swapper.py:20]: By changing it you lose little to know performance sr quality and it utilises the GPU properly

[the remark this answers is not in the document]
Comment (jasonkneen) [reply modules/processors/frame/face_swapper.py:20]: Sure

**Comment by leihuayi** [comment]
I was also thinking of making a PR for changing the link of model downloaded for GPU, but keep a fp16 version: `https://github.com/facefusion/facefusion-assets/releases/download/models/inswapper_128_fp16.onnx`
Because indeed the default link does not work (see issues  #717 , #697 ).

Are you sure not using fp16 does not slow down a lot the processing time ?

**Comment by rotatorotator** [comment]
Can you please explain why this is better for GPU? Shouldn´t a non FP16 be slower?

**Pull request closed without merging**